core/src/controllers/rows.js

```diff
@@ -396,7 +411,7 @@ const insertRowInTable = async (req, res, next) => {
     */
     res.status(400).json({
       message: error.message,
-      error: error
+      error: error,
```

Comment:
@AbegaM - it looks like your linter is appending commas to the last item in each list. This seems mistaken.